### PR TITLE
Activate clang travis

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

File: binds/python/bind_vasculature.cpp [hunk 1]
```diff
@@ -12,9 +12,9 @@
 #include <morphio/vasc/properties.h>
 
 namespace py = pybind11;
-using namespace py::literals;
 
-void bind_vasculature(py::module &m) {
+static void bind_vasculature(py::module &m) {
+    using namespace py::literals;
```
Comment (mgeplf) [inline binds/python/bind_vasculature.cpp]: why the change?

Comment (wizmer) [reply binds/python/bind_vasculature.cpp]: Because clang complains about using `using namespace` in a global scope.

Comment (wizmer) [reply binds/python/bind_vasculature.cpp]: And the `static` because it complains about the fact there is no prototype otherwise

Comment (mgeplf) [reply binds/python/bind_vasculature.cpp]: Doesn't that `using namespace` trigger this: https://abseil.io/tips/153 ? Or am I reading it wrong?

Comment (mgeplf) [reply binds/python/bind_vasculature.cpp]: I guess it only brings in `_a`, so no worries

Comment (wizmer) [reply binds/python/bind_vasculature.cpp]: or I can just bring _a with `using py::literals::operator""_a` ?

Comment (wizmer) [reply binds/python/bind_vasculature.cpp]: (this syntax is sick by the way, I had no idea you could do that)
